### PR TITLE
[storage] edits to blob, queue, and file docstrings

### PR DESCRIPTION
**Pull request by kristapratico**
#7785

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-file/azure/storage/file/_models.py [hunk 1]
```diff
@@ -397,15 +390,33 @@ class DirectoryProperties(DictMixin):
...
     :ivar str etag:
         The ETag contains a value that you can use to perform operations
         conditionally.
     :ivar bool server_encrypted:
         Whether encryption is enabled.
-    :ivar dict metadata: A dict with name_value pairs to associate with the
+    :keyword dict metadata: A dict with name_value pairs to associate with the
         directory as metadata.
+    :ivar change_time: Change time for the file.
+    :type change_time: str or ~datetime.datetime
```
Comment (annatisch) [inline sdk/storage/azure-storage-file/azure/storage/file/_models.py]: Should be "vartype"?

File: sdk/storage/azure-storage-file/azure/storage/file/_shared_access_signature.py [hunk 2]
```diff
@@ -351,10 +363,6 @@ def generate_share_sas(
         or address range specified on the SAS token, the request is not authenticated.
         For example, specifying sip=168.1.5.65 or sip=168.1.5.60-168.1.5.70 on the SAS
         restricts the request to those IP addresses.
-    :keyword str protocol:
-        Specifies the protocol permitted for a request made. Possible values are
-        both HTTPS and HTTP (https,http) or HTTPS only (https). The default value
-        is https,http. Note that HTTP only is not a permitted value.
```
Comment (annatisch) [inline sdk/storage/azure-storage-file/azure/storage/file/_shared_access_signature.py]: Why was this removed?

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_service_client_async.py [hunk 3]
```diff
@@ -417,10 +419,11 @@ def list_containers(
             The container to delete. This can either be the name of the container,
             or an instance of ContainerProperties.
         :type container: str or ~azure.storage.blob.ContainerProperties
-        :param ~azure.storage.blob.BlobLeaseClient lease:
+        :param lease:
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_service_client_async.py]: This should be `keyword` lease?

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py [hunk 4]
```diff
@@ -1411,25 +1423,24 @@ def __init__(
...
-            Must be set if length is provided.
+            Start of byte range to use for writing to a section of the blob.
             Pages must be aligned with 512-byte boundaries, the start offset
             must be a modulus of 512 and the length  must be a modulus of
             512.
         :param int length:
             Number of bytes to use for writing to a section of the blob.
             Pages must be aligned with 512-byte boundaries, the start offset
-            must be a modulus of 512 and the length must be a modulus of
+            must be a modulus of 512 and the end offset must be a modulus of
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py]: This should be "length"

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py [hunk 5]
```diff
@@ -1579,19 +1595,19 @@ def __init__(
...
+            Start of byte range to use for writing to a section of the blob.
             Pages must be aligned with 512-byte boundaries, the start offset
-            must be a modulus of 512.
+            must be a modulus of 512 and the length  must be a modulus of
+            512.
         :param int length:
             Number of bytes to use for writing to a section of the blob.
             Pages must be aligned with 512-byte boundaries, the start offset
-            must be a modulus of 512 and the length must be a modulus of
+            must be a modulus of 512 and the end offset must be a modulus of
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py]: Should be length

File: sdk/storage/azure-storage-file/azure/storage/file/_models.py (hunk 1 above)
Comment (kristapratico) [reply sdk/storage/azure-storage-file/azure/storage/file/_models.py]: Yes, good catch

File: sdk/storage/azure-storage-file/azure/storage/file/_shared_access_signature.py (hunk 2 above)
Comment (kristapratico) [reply sdk/storage/azure-storage-file/azure/storage/file/_shared_access_signature.py]: It was defined twice in the docstring

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_service_client_async.py (hunk 3 above)
Comment (kristapratico) [reply sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_service_client_async.py]: It's actually a parameter for delete_container()

File: sdk/storage/azure-storage-file/azure/storage/file/_models.py (hunk 1 above)
Comment (annatisch) [reply sdk/storage/azure-storage-file/azure/storage/file/_models.py]: This one is also on line 405 and 407

Comment (kristapratico) [reply sdk/storage/azure-storage-file/azure/storage/file/_models.py]: thanks, fixing
